apps/tangle-dapp/constants/polkadot.ts

```diff
@@ -142,3 +142,21 @@ export const getValidatorIdentity = async (
     );
   }
 };
+
+export const isNominatorAlreadyBonded = async (
```

Comment:
Does this really need a try/catch? It seems like you wrap it in a try/catch in the component the calls it.